tls/s2n_post_handshake.c

```diff
@@ -63,7 +63,8 @@ int s2n_post_handshake_send(struct s2n_connection *conn, s2n_blocked_status *blo
 
     GUARD(s2n_key_update_send(conn));
     GUARD(s2n_flush(conn, blocked));
-    GUARD(s2n_stuffer_rewrite(&conn->out));
+
+    S2N_RESULT_TO_POSIX(s2n_tls13_server_nst_send(conn, blocked));
```

Comment:
Since s2n_tls13_server_nst_send handles its own "s2n_flush", should "s2n_key_update_send" do the same?

Reply:
new_session_ticket needs to flush as it is potentially sending multiple new session ticket messages. Key update will only ever send one message. 🤷‍♀️ There is no need to change this imo.

Reply:
I think we should just have consistent behavior between the _send methods if possible. If nst_send handles flushing, then key_update_send probably should too. Is there any downside to moving the flush inside the send method?